actioncable/actioncable.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   }
 
   # NOTE: Please read our dependency guidelines before updating versions:
-  # https://edgeguides.rubyonrails.org/security.html#dependency-management-and-cves
+  # https://guides.rubyonrails.org/security.html#dependency-management-and-cves
```

Comment:
These gemspec changes were added in: https://github.com/rails/rails/pull/34392 and https://github.com/rails/rails/pull/34388. So at the time linking to edge guides made sense as the regular guides didn't have this section yet.